### ISSUE TITLE
Errors writing to State file

### ISSUE THREAD

**Issue by zrisher** [stackcollision/GardenConquest#53]
```
[2015-05-20 19:12:13,113][DEBUG][Conquest Core][StateTracker][saveState][][]Exception occured: System.IO.IOException: The process cannot access the file 'C:\Users\Zach\AppData\Roaming\SpaceEngineers\Storage\GardenConquestDistro_GardenConquest\GCState.xml' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at Sandbox.ModAPI.MyAPIUtilities.Sandbox.ModAPI.IMyUtilities.WriteFileInLocalStorage(String file, Type callingType)
   at GardenConquest.Core.StateTracker.saveState()
```

**Comment by zrisher** [comment]
Still occurring

```
015-06-02 10:58:47,968][DEBUG][Conquest Core][Server][saveTimer][][]Save timer triggered
[2015-06-02 10:58:47,968][DEBUG][Conquest Core][StateTracker][saveState][][]Saving state to file
[2015-06-02 10:58:47,969][DEBUG][Conquest Core][StateTracker][saveState][][]Write finished
[2015-06-02 11:03:47,968][DEBUG][Conquest Core][Server][saveTimer][][]Save timer triggered
[2015-06-02 11:03:47,968][DEBUG][Conquest Core][StateTracker][saveState][][]Saving state to file
[2015-06-02 11:03:47,968][DEBUG][Conquest Core][StateTracker][saveState][][]Exception occured: System.IO.IOException: The process cannot access the file 'C:\ProgramData\SpaceEngineersDedicated\HostASpace.com_216_448-628-Empire\Storage\450540708.sbm_GardenConquest\GCState.xml' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at Sandbox.ModAPI.MyAPIUtilities.Sandbox.ModAPI.IMyUtilities.WriteFileInLocalStorage(String file, Type callingType)
   at GardenConquest.Core.StateTracker.saveState()
```

**Comment by zrisher** [comment]
Fixed via #71
